### PR TITLE
AVM: Generate langspec for each version

### PR DESCRIPTION
**Pull request by jasonpaulos**
## Summary

Addresses #5131, which asks for a `langpsec.json` file to be generated for each TEAL version.

I've also generated the `TEAL_opcodes.md` file for each version, since I view this as the human-readable equivalent of `langspec.json`.

Now the files `langpsec_v%d.json` and `TEAL_opcodes_v%d.md` will be generated for each version.

## Test Plan

Documentation-only change

### CODE REVIEW

File: data/transactions/logic/TEAL_opcodes_v8.md [hunk 1]
```diff
@@ -14,27 +14,21 @@ Ops have a 'cost' of 1 unless otherwise specified.
 - Bytecode: 0x01
 - Stack: ..., A: []byte &rarr; ..., [32]byte
 - SHA256 hash of value A, yields [32]byte
-- **Cost**:
-    - 7 (v1)
-    - 35 (since v2)
+- **Cost**: 35
```
Comment (jannotti) [inline data/transactions/logic/TEAL_opcodes_v8.md:17]: I like this. No point in reminding people about old versions that had miscalculated costs!

**Review by jannotti — COMMENTED** [review]
We'll have to coordinate with the process that gets the docs onto the website, since there is no `TEAL_opcodes.md` any more.  I imagine we either want to create it, as a copy of the last version or (I think better) the process that copies them into place should copy and link to them all. @JasonWeathersby

**Comment by jasonpaulos** [comment]
Created https://github.com/algorand/docs/pull/1156 to update the docs site with this change.

File: cmd/opdoc/opdoc.go [hunk 2]
```diff
@@ -342,7 +329,7 @@ func (nt namedType) boundString() string {
 
 // LanguageSpec records the ops of the language at some version
 type LanguageSpec struct {
-	EvalMaxVersion  int
+	EvalMaxVersion  uint64
```
Comment (jannotti) [inline cmd/opdoc/opdoc.go:332]: Maybe this should be renamed to just "Version" now?

**Review by jannotti — APPROVED** [review]
Looks good to me. I'd like to get this merged soon, so I can work out how it changes the process of giving spec changes to the Foundation. (probably pretty minimal)

**Review by bbroder-algo — APPROVED** [review]

File: cmd/opdoc/opdoc.go (hunk 2 above)
Comment (jasonpaulos) [reply cmd/opdoc/opdoc.go]: Good idea, done in 780068665

**Review by jannotti — DISMISSED** [review]

**Review by bbroder-algo — APPROVED** [review]

**Review by winder — APPROVED** [review]

**Pull request merged**